src/main.zig

```diff
@@ -439,8 +438,6 @@ const usage_build_generic =
     \\  -fno-llvm                 Prevent using LLVM as the codegen backend
     \\  -flibllvm                 Force using the LLVM API in the codegen backend
     \\  -fno-libllvm              Prevent using the LLVM API in the codegen backend
-    \\  -fclang                   Force using Clang as the C/C++ compilation backend
-    \\  -fno-clang                Prevent using Clang as the C/C++ compilation backend
```

Comment:
I think using `CFrontend` internally is a good idea, however, as for the CLI, I don't think zig ever plans to have more options for C frontends. The plan is:

* (now) support clang and aro
* (future) support aro only

I think `-fclang` and `-fno-clang` is nice since it is very similar to the llvm and lld flags above, and all 3 sets of flags will disappear together one day.

Do you have a strong preference for `-c-frontend`? I would prefer keeping the CLI as `-fclang` and `-fno-clang`.

Reply:
No preference, `-fclang`/`-fno-clang` works for me.

Reply:
Nice! In that case, feel free to proceed with the merge after making that change :+1: